tap_xero/client.py

```diff
@@ -12,11 +12,13 @@
 import pytz
 import backoff
 import singer
+from requests.exceptions import Timeout, ConnectTimeout
```

Comment:
Is there any specific reason to import ConnectTimeout?

Reply:
> Is there any specific reason to import ConnectTimeout?

It was not used anywhere as ConnectTimeout inherits Timeout, hence removed it